pkg/bundle/source.go

```diff
@@ -343,21 +344,37 @@ func (b *bundleData) populateData(bundles []string, formats *trustapi.Additional
 	return nil
 }
 
-// remove duplicate certificates from bundles
-func deduplicateBundles(bundles []string) ([]string, error) {
+// ByHash is a type alias for sorting the slice of [32]byte arrays
+type ByHash [][32]byte
```

Comment:
```suggestion
type byHash [][32]byte
```

I don't like this type, but if we decide to keep it, I don't think it needs to be exported.

Reply:
done